packages/swing-store/src/swingStore.js

```diff
@@ -549,38 +565,38 @@ export function makeSwingStore(dirPath, forceReset, options = {}) {
     return db;
   }
 
-  const transcriptStorePublic = {
-    initTranscript: transcriptStore.initTranscript,
-    rolloverSpan: transcriptStore.rolloverSpan,
-    rolloverIncarnation: transcriptStore.rolloverIncarnation,
-    getCurrentSpanBounds: transcriptStore.getCurrentSpanBounds,
-    addItem: transcriptStore.addItem,
-    readSpan: transcriptStore.readSpan,
-    stopUsingTranscript: transcriptStore.stopUsingTranscript,
-    deleteVatTranscripts: transcriptStore.deleteVatTranscripts,
-  };
+  const transcriptStore = attenuate(transcriptStoreInternal, {
+    initTranscript: 'pick',
```

Comment:
`pick` is misleading because it suggests a label like `skip` would skip but it also permits.

Any reason not to require this to be boolean?

Reply:
Compatibility with https://github.com/Agoric/agoric-sdk/blob/8ef3fde4fbbca626c84b31c1d65ad676942c9e20/packages/vats/src/core/utils.js#L123 , which should be reimplemented on top of this.